src/apps/loan-list/materials/stackable-material.tsx

```diff
@@ -22,7 +26,9 @@ const StackableMaterial: FC<StackableMaterialProps & MaterialProps> = ({
 }) => {
   const t = useText();
   const dispatch = useDispatch();
-
+  const [additionalMaterials] = useState(
+    amountOfMaterialsWithDueDate ? amountOfMaterialsWithDueDate - 1 : 0
```

Comment:
I haven't see useState been used without a getter before. But I guess it can be useful :)